src/lib/blueprint.ts

```diff
@@ -117,14 +121,327 @@ interface ObjectProperty extends BaseProperty {
+    resources: createResources(
+      openapi.components.schemas,
+      isFakeData,
+      targetSchema,
+    ),
+  }
+}
+
+const createRoutes = (
+  paths: OpenAPIPaths,
+  isFakeData: boolean,
+  targetPath: string,
+): Route[] => {
+  return Object.entries(paths)
+    .filter(([path]) => isFakeData || path === targetPath)
+    .map(([path, pathItem]) => createRoute(path, pathItem))
+}
+
+const createRoute = (path: string, pathItem: OpenAPIPathItem): Route => {
+  const pathParts = path.split('/')
+  const routePath = `/${pathParts.slice(1, -1).join('/')}`
+
+  return {
+    path: routePath,
+    namespace: { path: `/${pathParts[1]}` },
+    endpoints: createEndpoints(path, pathItem),
+    subroutes: [],
+  }
+}
+
+const createEndpoints = (
+  path: string,
+  pathItem: OpenAPIPathItem,
+): Endpoint[] => {
+  return Object.entries(pathItem)
+    .filter(
+      ([, operation]) => typeof operation === 'object' && operation !== null,
+    )
+    .map(([method, operation]) =>
+      createEndpoint(method, operation as OpenAPIOperation, path),
+    )
+}
+
+const createEndpoint = (
+  method: string,
+  operation: OpenAPIOperation,
+  path: string,
+): Endpoint => {
+  const pathParts = path.split('/')
+  const endpointPath = `/${pathParts.slice(1, -1).join('/')}`
+
+  return {
+    name:
+      'operationId' in operation && typeof operation.operationId === 'string'
+        ? operation.operationId
+        : `${path.replace(/\//g, '')}${method.charAt(0).toUpperCase() + method.slice(1).toLowerCase()}`,
+    path: endpointPath,
+    description:
+      'description' in operation && typeof operation.description === 'string'
+        ? operation.description
+        : '',
+    isUndocumented: false,
+    isDeprecated: false,
+    deprecationMessage: '',
+    parameters: createParameters(operation),
+    request: createRequest(method, operation),
+    response: createResponse(
+      'responses' in operation ? operation.responses : {},
+    ),
+  }
+}
+
+const createParameters = (operation: OpenAPIOperation): Parameter[] => {
+  return 'parameters' in operation && Array.isArray(operation.parameters)
+    ? operation.parameters
+        .filter((param) => typeof param === 'object' && param !== null)
+        .map(createParameter)
+    : []
+}
+
+const createParameter = (param: OpenAPIParameter): Parameter => {
+  return {
+    name: 'name' in param && typeof param.name === 'string' ? param.name : '',
+    isRequired:
+      'required' in param && typeof param.required === 'boolean'
+        ? param.required
+        : false,
+    isUndocumented: false,
+    isDeprecated: false,
+    deprecationMessage: '',
+    description:
+      'description' in param && typeof param.description === 'string'
+        ? param.description
+        : '',
+  }
+}
+
+const createRequest = (
+  method: string,
+  operation: OpenAPIOperation,
+): Request => {
+  const uppercaseMethod = openapiMethodToMethod(
+    method.toLowerCase() as LowercaseMethod,
```

Comment:
I'm still confused, if the method is not already lower case then why do we need this function?
```suggestion
    method
```